model/SearchEngine/Contract/IndexerInterface.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
```

Comment:
Suggesting to quickly fix this small PSR issue as part of this PR

```suggestion
<?php

/**
```